doc/classes/@GlobalScope.xml

```diff
@@ -1330,6 +1330,36 @@
 		<constant name="VALIGN_BOTTOM" value="2" enum="VAlign">
 			Vertical bottom alignment, usually for text-derived classes.
 		</constant>
+		<constant name="INLINE_ALIGN_TOP_TO" value="0" enum="InlineAlign">
+			Aligns top of the inline object (e.g. image, table) to the position of specified by [code]INLINE_ALIGN_TO_*[/code] constant.
```

Comment:
"to the position of specified by ... constant" - there's either a missing word ("position of **the text** specified by **a** ... constant") or one word too many ("of").